nodejs/week3/app.js

```diff
@@ -0,0 +1,48 @@
+// @ts-nocheck
+const knex = require("knex")({
+    client: "mysql2",
+    connection: {
+      host: process.env.DB_HOST || "127.0.0.1",
+      port: process.env.DB_PORT || 33061,
+      user: process.env.DB_USER || "root",
+      password: process.env.DB_PASSWORD || "mypassword",
+      database: process.env.DB_NAME || "hyf_node_week3_warmup",
+      multipleStatements: true,
+    },
+  })
+  
+  const express = require("express")
+  const app = express()
+  const port = process.env.PORT || 3000
```

Comment:
You should run a code formatter because the indentation here and in the rest of file looks a bit off.